specification/trace/semantic_conventions/messaging.md

```diff
@@ -82,45 +83,42 @@ Given these definitions, the remainder of this section describes the semantic co
 
 ### Context propagation
 
-A message may pass many different components and layers in one or more intermediaries
+A message may traverse many different components and layers in one or more intermediaries
 when it is propagated from the producer to the consumer(s). To be able to correlate
 consumer traces with producer traces using the existing context propagation mechanisms,
 all components must propagate context down the chain.
 
-This imposes a difficulty because it cannot be assumed, and in many cases,
-it is not even desired, that all components and layers to be instrumented and
-propagate context according to OpenTelemetry requirements.
+This imposes a requirement that all components are instrumented
```

Comment:
I am reluctant about the `This` here. You can't read this paragraph without the one before it. Maybe something like: The anatomy of systems relying on messaging imposes a requirement...?

Also imposes a *requirement* ... according to OpenTelemetry *requirements*.

Reply:
Agreed. I changed it in [16bbdc7](https://github.com/dynatrace-oss-contrib/opentelemetry-specification/pull/22/commits/16bbdc7d4447813da7e3209cd47437700fcd4a65).